main/lsp/json_types.cc

```diff
@@ -151,11 +151,4 @@ unique_ptr<Position> Position::copy() const {
     return make_unique<Position>(line, character);
 }
 
-void SorbetWorkspaceEditCounts::merge(const SorbetWorkspaceEditCounts &other) {
```

Comment:
No longer necessary. I can determine how many edits are in a single `LSPFileUpdate` by subtracting the versions it contains.